.github/workflows/deploy.yml

```diff
@@ -70,15 +70,7 @@ jobs:
         cf_password: ${{ secrets.CLOUDGOV_PASSWORD }}
         cf_org: gsa-tts-benefits-studio
         cf_space: notify-staging
-        push_arguments: >-
-          --vars-file deploy-config/staging.yml
-          --var DANGEROUS_SALT="$DANGEROUS_SALT"
-          --var SECRET_KEY="$SECRET_KEY"
-          --var ADMIN_CLIENT_SECRET="$ADMIN_CLIENT_SECRET"
-          --var NEW_RELIC_LICENSE_KEY="$NEW_RELIC_LICENSE_KEY"
-          --var NOTIFY_E2E_TEST_EMAIL="$NOTIFY_E2E_TEST_EMAIL"
-          --var NOTIFY_E2E_TEST_PASSWORD="$NOTIFY_E2E_TEST_PASSWORD"
-          --var LOGIN_DOT_GOV_REGISTRATION_URL="$LOGIN_DOT_GOV_REGISTRATION_URL"
+        cf_command: "push -f manifest.yml --vars-file deploy-config/staging.yml --var var-name=${{ secrets.DANGEROUS_SALT }} --var var-name=${{ secrets.SECRET_KEY }} --var var-name=${{ secrets.ADMIN_CLIENT_SECRET }} --var var-name=${{ secrets.NEW_RELIC_LICENSE_KEY }} --var var-name=${{ secrets.NOTIFY_E2E_TEST_EMAIL }} --var var-name=${{ secrets.NOTIFY_E2E_TEST_PASSWORD }} --var var-name=${{ secrets.LOGIN_DOT_GOV_REGISTRATION_URL }} --strategy rolling"
```

Comment:
I think this is right for the most part, at least in terms of converting it to a full command!

One thing though, you didn't have to refer to things like `${{ secrets.DANGEROUS_SALT }}` directly - this is because they're already loaded in as environment variables to this container environment just a bit further up (lines 60 - 66).  You could go back to just referring to them as environment variables themselves to help make this slightly easier to read. 🙂